src/common/actions/auth-store.js

```diff
@@ -251,15 +253,15 @@ async function fetchAccountInfo(root, discharge) {
 async function setShortNamespace(root, discharge, userName) {
   // Try setting the short namespace to the SSO username.  This may not
   // work, but it's the best we can do automatically.
-  const response = await fetch(`${BASE_URL}/api/store/account`, {
+  const response = await fetch(`${STORE_API_URL}/account`, {
     method: 'PATCH',
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify({
-      short_namespace: userName,
-      root,
-      discharge
-    })
+    headers: {
+      'Authorization': `Macaroon root="${root}", discharge="${discharge}"`,
+      'Content-Type': 'application/json'
```

Comment:
1 reason to do it might be that Macaroon auth bugs are pretty awful to debug, and mitigating a typo here would defend against someone having that journey.